dockerfiles/hpcc/values.yaml

```diff
@@ -13,8 +29,9 @@ global:
     ##
     storageClass: ""
     # existingClaim: ""
```

Comment:
Would benefit from documenting existingClaim and forcePermissions options.

Reply:
have added some comments in new PR